doc/install.txt

```diff
@@ -135,7 +135,7 @@ Manual setup for OSX systems
    Now install py2app:
    $ sudo easy_install -U py2app
 
-5. git clone git@github.com:linkchecker/linkchecker.git linkchecker
+5. git clone https://github.com/linkchecker/linkchecker.git
```

Comment:
```suggestion
5. git clone https://github.com/linkchecker/linkchecker
```

I mean, either works, but the version without .git can also be opened in a web browser.

Well, both can be opened in a web browser, but the .git-less version is shorter and prettier.